assignments/callbacks.js

```diff
@@ -27,24 +27,46 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
+
 }
+getLength(items, function (element) {
+  console.log(element)
+})
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length - 1]);
 }
+last(items, function (food) {
+  console.log(food)
+})
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
+sumNums(3, 5, function (items) {
+  console.log(items)
+})
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y)
 }
+multiplyNums(3, 5, function (items) {
```

Comment:
great use of the cb, but you could switch it if you wanted and make the cb the use case and the HOF the return